cmake/onnxruntime_python_interface.cmake

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+file(GLOB onnxruntime_python_interface_cc_srcs
+"${ONNXRUNTIME_ROOT}/core/dlpack/dlpack_python.cc"
+"${ONNXRUNTIME_ROOT}/core/dlpack/dlpack_python.h"
+"${ONNXRUNTIME_ROOT}/core/dlpack/python_common.h"
+)
+
+onnxruntime_add_static_library(onnxruntime_python_interface ${onnxruntime_python_interface_cc_srcs})
```

Comment:
First, you don't need to use GLOB. As the list of file names do not contain "*".

So you can change it to 
```cmake
set(onnxruntime_python_interface_cc_srcs
"${ONNXRUNTIME_ROOT}/core/dlpack/dlpack_python.cc"
"${ONNXRUNTIME_ROOT}/core/dlpack/dlpack_python.h"
"${ONNXRUNTIME_ROOT}/core/dlpack/python_common.h"
)

onnxruntime_add_static_library(onnxruntime_python_interface ${onnxruntime_python_interface_cc_srcs})
```

Furthermore,  it could be even simpler
```cmake
onnxruntime_add_static_library(onnxruntime_python_interface "${ONNXRUNTIME_ROOT}/core/dlpack/dlpack_python.cc"
"${ONNXRUNTIME_ROOT}/core/dlpack/dlpack_python.h"
"${ONNXRUNTIME_ROOT}/core/dlpack/python_common.h")
```